docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -236,13 +240,13 @@ It's also important to follow the [steps described here](https://learn.microsoft
+| ----------- | -------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity    | **Microsoft Entra ID:** `https://<IDENTITY_URL>/auth/login-callback` <br/><br/> **Helm:** `https://<IDENTITY_URL>`         |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Nbs: On the web page, these values sometimes break onto the next line if longer, but sometimes don't - you could standardise this for all of them by adding a <br/> after before each value, so they are always on the next line?
e.g. **Microsoft Entra ID:**<br/>`https://<OPERATE_URL>/identity-callback`